samples/built-in-policy/app-service-audit-diagnostic-logs/azurepolicy.json

```diff
@@ -0,0 +1,57 @@
+{
+    "properties": {
+        "displayName": "Audit enabling of diagnostic logs in App Services",
+        "policyType": "BuiltIn",
+        "mode": "All",
+        "description": "Audit enabling of diagnostic logs on the app. This enables you to recreate activity trails for investigation purposes if a security incident occurs or your network is compromised",
+        "metadata": {
+            "category": "App Service"
+        },
+        "parameters": { 
+            "effect": { 
+                "type": "string",
+                "defaultValue": "Audit",
+                "allowedValues": ["Audit", "Disabled"],
+                "metadata": {
+                    "displayName": "Effect",
+                    "description": "Enable or disable the execution of the policy"
+                }
+            } 
+        },
+        "policyRule": {
+            "if": {
+                "allOf": [
+                {
+                   "field": "type",
+                   "equals": "Microsoft.Web/sites/config"
+                },
+                {
+                   "field": "name",
+                   "equals": "web"
+                },
+                {
+                   "anyOf": [
+                      {
+                         "field": "Microsoft.Web/sites/config/detailedErrorLoggingEnabled",
+                         "notEquals": "true"
+                      },
+                      {
+                         "field": "Microsoft.Web/sites/config/httpLoggingEnabled",
+                         "notEquals": "true"
+                      },
+                      {
+                         "field": "Microsoft.Web/sites/config/requestTracingEnabled",
+                         "notEquals": "true"
+                      }
+                   ]
+                }
+                ]
+            },
+            "then": {
+                "effect": "[parameters('effect')]"
+            }
+        }
+    },
+    "id": "/providers/Microsoft.Authorization/policyDefinitions/752c6934-9bcc-4749-b004-655e676ae2ac",
+    "name": "752c6934-9bcc-4749-b004-655e676ae2ac"
```

Comment:
`id` should be removed from all the policies. Customers cannot create new policies at the scope `/providers/Microsoft.Authorization/` (it's reserved for built-ins), so leaving it here would block the samples from working.

`name` should also be removed (it's optional - that GUID is auto-generated), or replaced with a meaningful name